docs/CLI-INSTALLATION.md

```diff
@@ -0,0 +1,67 @@
+macOS and GNU/Linux
```

Comment:
Afraid I'm not a markdown / html expert - what do I need to change, and where? ;)